website/docs/basics/blocks-intro.md

```diff
@@ -195,4 +195,32 @@ And folder structure `src/Blocks/blocks` is kinda weird.
 
 You can, but you shouldn't. We use this key to provide the CSS selectors on all our custom blocks in the block editor. Additional styles are added to this selector for the wrapper to work on full width and remove some of the native block editor styles. 
 
-So don't remove or change this 😅 .
+So don't remove or change this 😅.
```

Comment:
Can't leave a comment on that line but there's a heading right above it:

> ### Why is this my blocks folder called custom and not blocks, for example?

I'd change that to: 

> ### Why is my blocks folder called `custom` and not `blocks`, for example?